### ISSUE TITLE
Cannot open items or categories imported from Joomla - Also cannot add new items or categories

### ISSUE THREAD

**Issue by TGoodrich** [getk2/k2-v3-dev-build#234]
Fresh Joomla 3.4.3 install and latest K2 Dev v3. Also using the latest Gantry 5 and Hydrogen theme from Rockettheme.

No K2 errors when installed but when clicking on a item or category in admin interface a red banner with an exclamation mark appears above the items and categories. No message appears in the banner and nothing else happens, it just stays on the same page.

When clicking the "+ Add" button for items and categories nothing happens.

**Comment by kavadas** [comment]
Hi,

Do you see any errors in the browser console? Can you also try the same without Gantry?

**Comment by TGoodrich** [comment]
Sorry this reply is a long time coming... I tried the latest build and had the same issue as before. After i decided to completely uninstall my joomla installation and start from scratch. Even though I have not had time to completely test v3 I can confirm I can now add new categories and items. I have also reinstalled gantry 5 with hydrogen theme so that was not the problem. I will start adding extensions back in and report back if a particular extension causes issues.
